### PR TITLE
Fix diagnotic topics to be properly namespaced

### PR DESCRIPTION
**Pull request by Timple**
Let's clean this up for better debugging. 

Untested

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Is this ready to be reviewed?

**Comment by Timple** [comment]
Ready to be tested

File: parameters/hardware/actuators/base_controller_vel.ops [hunk 1]
```diff
@@ -178,7 +178,7 @@ stream ("BASE_OdometryPublisher.reset", 		ros.topic(ROBOT+"/base/reset_odometry"
 connect ("BASE_ReadReferences.out", 			"BASE_Diagnostics.vec1", 		ConnPolicy() )
 connect ("BASE_Controller.jointErrors", 		"BASE_Diagnostics.vec2", 		ConnPolicy() )
 connect ("BASE_SI2Volt.out", 					"BASE_Diagnostics.vec3", 		ConnPolicy() )
-stream ("BASE_Diagnostics.diagnostics", 		ros.topic(ROBOT+"/base_diagnostics"));
+stream ("BASE_Diagnostics.diagnostics", 		ros.topic(ROBOT+"/diagnostics"));
```
Comment (MatthijsBurgh) [inline parameters/hardware/actuators/base_controller_vel.ops]: ROBOTNS is used in other OPS files

**Comment by Timple** [comment]
Beter zo @MatthijsBurgh?

**Comment by MatthijsBurgh** [comment]
Jep

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**